install.sh

```diff
@@ -20,23 +20,41 @@ fi
 #-----------------------
 
 #### Install Packages
-$SUDO apt-get update && $SUDO apt-get install -yq \
+$SUDO apt-get update && $SUDO apt-get install -yq libboost-container-dev
+$SUDO apt-get install -yq xfce4
+$SUDO apt-get install -yq \
     python3 \
     python3-pip \
     python3-venv \
     g++\
     cmake \
-    libboost-container-dev \
+    libboost-all-dev \
     graphviz \
     gnuplot \
     curl \
     xvfb \
+    libx11-dev \
+    xorg-dev \
+    libglu1-mesa-dev \
+    freeglut3-dev \
+    libglew1.5 \
+    libglew1.5-dev \
+    libglu1-mesa \
+    libglu1-mesa-dev \
+    libgl1-mesa-glx \
+    libgl1-mesa-dev \
+    lp-solve \
+    liblpsolve55-dev \
+    libgtest-dev \
 &&  $SUDO apt-get clean
 
 #### Install klayout 
 curl -o ./klayout_0.26.3-1_amd64.deb https://www.klayout.org/downloads/Ubuntu-18/klayout_0.26.3-1_amd64.deb
```

Comment:
We should probably update klayout.

Reply:
Yes, this way it downloads both a version for Ubuntu 18 and a version for Ubuntu 20, and preferably installs the newer version.